Helper/Order.php

```diff
@@ -95,6 +97,7 @@ class Order extends AbstractHelper
             self::TS_ZERO_AMOUNT
         ],
         self::TS_AUTHORIZED => [
+            self::TS_AUTHORIZED,
```

Comment:
Does this mean that partially captured transaction when fetched still has the type of **cc_payment** and status **authorized**?

Reply:
Yes, after a partial capture, the transaction still has an authorized status. When I implemented multiple capture on M1, it had a completed status. I think I need to recheck it on M1 and put in a fix for it as there is new logic now.